.github/pull_request_template.md

```diff
@@ -34,5 +27,39 @@ _Please delete options that are not relevant._
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have created a helpful and easy to understand `README.md`
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
-- [ ] My changes generate no new warnings
-- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
+- [ ] I have added the project meta data in the PR template.
```

Comment:
Just one more checklist item: I have created the `requirements.txt` file if needed.